routing/localchans/manager.go

```diff
@@ -34,6 +35,9 @@ type Manager struct {
 	FetchChannel func(chanPoint wire.OutPoint) (*channeldb.OpenChannel,
 		error)
 
+	// HasActiveLike is used to check whether a channel is active or not.
+	HasActiveLink func(chanID lnwire.ChannelID) bool
```

Comment:
nit: would `IsActive` be a better name?

Reply:
Since I'm referencing https://github.com/lightningnetwork/lnd/blob/ebabda671786486d37a76718961672eabfa7b368/htlcswitch/switch.go#L2061 I though I might as well use the same name.

Reply:
I think HasActiveLink is good name because it returns true if there is a link in the link index *AND* the link is eligible to forward. Not to be confused by the active status of the channel.

Reply:
@carlaKC I'm changing my mind... changing it to IsChannelActive, just like we call it in the invoice sub-server. Thanks for mentioning it.